src/org/loklak/harvester/Post.java

```diff
@@ -35,8 +37,16 @@ public void setTimestamp() {
         this.setTimestamp(timestamp);
     }
 
-    //public abstract void getPostId();
+    public Date getTimestampDate() {
+        return new Date(this.timestamp);
+    }
+
+    //TODO: Set up TwitterTweet before setting this as abstract
```

Comment:
I have inherited Post here -> https://github.com/loklak/loklak_server/pull/1249/files#diff-c7c5000d9cf0abf8e64f6de0914d35d3
If I declare methods abstract in `Post` right now, everywhere where AbstractObjectEntry is inherited will have to define them. And not all of those child classes are scrapers